dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -18,6 +18,7 @@
 		cpu@0 {
 			device_type = "cpu";
 			compatible = "arm,cortex-m0";
+			clock-frequency = <48000000>;
```

Comment:
Clock frequency setting is done at board level as it depends on the clock generator selected/available on the board. 
So this should be removed from here.
Also, if `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC` is not used on STM32, this is not needed and can be removed with no impact.

Reply:
Fixed.